src/projects/projects.service.ts

```diff
@@ -1,26 +1,76 @@
 import { Injectable } from '@nestjs/common';
 import { CreateProjectDto } from './dto/create-project.dto';
-import { UpdateProjectDto } from './dto/update-project.dto';
+import { Model } from 'mongoose';
+import { InjectModel } from '@nestjs/mongoose';
+import { Projects } from './interfaces/projects.interface';
 
 @Injectable()
 export class ProjectsService {
-  create(createProjectDto: CreateProjectDto) {
-    return 'This action adds a new project';
-  }
+  constructor(@InjectModel('Projects') private projectsModel: Model<Projects>) {}
 
-  findAll() {
-    return `This action returns all projects`;
+  async create(userId: number, createProjectDto: CreateProjectDto): Promise<Projects> {
+    const user = await this.projectsModel.find({ userId: userId}).exec();
+    // If user not exist create new record
+    if (!user[0]){
+      const newProject = await new this.projectsModel(createProjectDto).save();
+      return newProject;
+    // If user exist create new record in user's projects
+    } else {
+      const newProject = await this.projectsModel
+      .findOne({ userId: userId })
+      .exec();
+      newProject.projects.push(createProjectDto.projects[0]);
+      return newProject.save();
+    }
   }
 
-  findOne(id: number) {
-    return `This action returns a #${id} project`;
+  // Get all projects for user
+  async findAll(userId: number): Promise<Projects[]> {
+    const projects = await this.projectsModel.find({ userId: userId }).exec();
+    return projects;
   }
 
-  update(id: number, updateProjectDto: UpdateProjectDto) {
-    return `This action updates a #${id} project`;
+  // Get project with a #${id}
+  async findOne(userId: number, projectId: number): Promise<Projects> {
+    const project = await this.projectsModel
+      .findOne({ userId: userId })
+      .exec();
+    const newProject = project.projects.filter((el) => el._id === projectId);
+    project.projects.splice(0, project.projects.length);
+    project.projects.push(...newProject);
+    return project;
+  }
+  // Update one of user's project where _id equal to projectId
+  async update(userId, projectId: number, createProjectDto: CreateProjectDto): Promise<Projects> {
+    const updatedProject = await this.projectsModel
```

Comment:
Same here. Remove redundant comments and console logs. There should be logs in case something went wrong.